tests/integration_tests/tests/utils.py

```diff
@@ -199,12 +198,5 @@ def patch_yaml(yaml_path, is_json=False, default_flow_style=True):
         yield patch
 
 
-def get_remote_storage_manager():
```

Comment:
Was never being used, and would pose a problem anyway (because of the open SQL session).